packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -92,11 +92,27 @@ export const RemoteScreenShare = React.memo(
       return (
         <VideoTile
           renderElement={
+            // Add the layer on top of the pptlive stream to diable keyboard event
+            // TODO need to remove after we have navigation feature ready
             renderElement ? (
-              <StreamMedia
-                videoStreamElement={renderElement}
-                loadingState={isReceiving === false ? 'loading' : 'none'}
-              />
+              <div style={{ top: 0, left: 0, width: '100%', height: '100%' }}>
+                <StreamMedia
+                  videoStreamElement={renderElement}
+                  loadingState={isReceiving === false ? 'loading' : 'none'}
+                />
+                <div
+                  style={{
+                    position: 'absolute',
+                    top: 0,
+                    left: 0,
+                    width: '100%',
+                    height: '100%',
+                    backgroundColor: 'transparent',
+                    pointerEvents: 'none',
```

Comment:
Was the decision here to also block out the ability to action links in the powerpoint? I know that we will get A11y bugs if we do this.

Reply:
This is a good point, should we be only capturing keyboard events?
Although another a11y point - if we can click on links inside PPTLive, we should be able to keyboard navigate to them. In which case we should only be disabling the events that trigger shifting the slide